configs/webpack.docs.dev.config.js

```diff
@@ -177,6 +177,56 @@ module.exports = {
             to: path.join(process.cwd(), 'assets', 'css')
         }]),
 
+        new CopyWebpackPlugin([{
+            from: path.join(process.cwd(), 'docs', 'app', 'showcase', 'list_view', 'dist'),
+            to: path.join(process.cwd(), 'dist', 'docs', 'showcase', 'list_view', 'dist')
+        }]),
+
+        new CopyWebpackPlugin([{
+            from: path.join(process.cwd(), 'docs', 'app', 'showcase', 'list_view', 'bower_components'),
+            to: path.join(process.cwd(), 'dist', 'docs', 'showcase', 'list_view', 'bower_components')
+        }]),
+
+        new CopyWebpackPlugin([{
+            from: path.join(process.cwd(), 'dist', 'ng1'),
```

Comment:
There is quite a lot of copying folders from the dist folder. Could we copy the entire dist folder over (excluding the dist/docs folder)? I know some folder names are different eg, ng1 instead of js, but we should just be able to update the paths in the showcase apps